bridges/otelzap/core.go

```diff
@@ -145,7 +158,11 @@ func (o *Core) Write(ent zapcore.Entry, fields []zapcore.Field) error {
 
 	if len(fields) > 0 {
 		attrbuf := convertField(fields)
+		attrbuf = slices.Grow(attrbuf, len(o.attr))
+		attrbuf = append(attrbuf, o.attr...)
```

Comment:
This may not need to allocate space if `r.AddAttributes` can fit `o.attr` and the fields into its "front" storage.

Instead, maybe, do something like this?

```go
r.AddAttributes(o.attr...)
if len(fields) > 0 {
	attrbuf := convertField(fields)
	r.addAttributes(attrbuf...)
}